src/styles/_helpers.scss

```
@@ -32,11 +25,45 @@
 }
 
 @mixin activeContent() {
-  &--active {
+  &:active, &--active {
     @content
   }
 }
 
-@function borderColor($hover: false) {
-  @return rgba($white, if($hover, 0.2, 0.1))
+@function borderColor($interacted: false) {
+  @return rgba($white, if($interacted == true, .2, .1))
+}
+
+@mixin border($interact: true) {
+
+  border: 1px solid borderColor();
+
+  @if ($interact == true) {
```

Does this need `== true`?

```suggestion
  @if ($interact) {
```